lib/mutant/ast/meta.rb

```diff
@@ -81,7 +44,8 @@ def assignment?
         # @api private
         #
         def attribute_assignment?
-          arguments.one? && attribute_assignment_selector?
+          !Types::METHOD_OPERATORS.include?(selector) &&
+          selector.to_s.end_with?(ATTRIBUTE_ASSIGNMENT_SELECTOR_SUFFIX)
```

Comment:
@mbj if so, is there a test for this? I didn't see anything when I looked for it in this diff (it could very well be covered elsewhere by an earlier commit)